scripts/Muon/GUI/Common/utilities/algorithm_utils.py

```diff
@@ -20,119 +20,120 @@ def run_MuonPreProcess(parameter_dict):
     """
     alg = mantid.AlgorithmManager.create("MuonPreProcess")
     alg.initialize()
-    alg.setAlwaysStoreInADS(False)
-    alg.setProperty("OutputWorkspace", "__notUsed")
+    alg.setAlwaysStoreInADS(True)
+    alg.setProperty("OutputWorkspace", "__pre_processed_data")
     alg.setProperties(parameter_dict)
     alg.execute()
-    return alg.getProperty("OutputWorkspace").value
+    return "__pre_processed_data"
```

Comment:
looks like all of the comments will be out of date, as we now return a string/name of the workspace.